ibis/pandas/api.py

```diff
@@ -1,9 +1,36 @@
 from __future__ import absolute_import
 
+import ibis
+import toolz
+
 from ibis.pandas.client import PandasClient
 from ibis.pandas.decimal import execute_node  # noqa: F401
 from ibis.pandas.execution import execute  # noqa: F401
 
 
+__api__ = 'connect', 'execute'
```

Comment:
This should be `__all__`

Reply:
Does this need to be a list?

Reply:
It can be, but I'm going to leave it as a `tuple` (after renaming to `__all__`).